src/integrationTest/java/uk/gov/hmcts/reform/cdapi/CdAuthorizationEnabledIntegrationTest.java

```diff
@@ -85,21 +88,21 @@ public void setUpIdamStubs() throws Exception {
                                                .withHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
                                                .withBody("rd_commondata_api")));
 
+
+        UserInfo userDetails = UserInfo.builder()
+            .uid("%s")
+            .givenName("Super")
+            .familyName("User")
+            .roles(List.of("%s"))
+            .build();
+
```

Comment:
where are email, accountStatus and roles?

Reply:
Team has used UserInfo for inbuild jar Idam-client-2.0.0 and it is using only these parameters for the connection.